### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by Anna-Shapovalova — APPROVED** [review]
GJ! 👍 Left some comments)

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
```
[the remark this answers is not in the document]
Comment (illiarerun) [reply src/components/TodoInfo/TodoInfo.tsx:20]: agree, fix it

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,4 +1,16 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todo: Array<Todo>;
```
[the remark this answers is not in the document]
Comment (illiarerun) [reply src/components/TodoList/TodoList.tsx:5]: agree

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+};
+
+export const TodoList: React.FC<Props> = ({ todo }) => (
+  <section className="TodoList">
+    {todo.map(item => (
```
[the remark this answers is not in the document]
Comment (illiarerun) [reply src/components/TodoList/TodoList.tsx:10]: agree

File: src/components/TodoList/TodoList.tsx [hunk 4]
```diff
@@ -1,4 +1,16 @@
...
+      <>
+        <TodoInfo key={item.id} todo={item} />
+      </>
```
[the remark this answers is not in the document]
Comment (illiarerun) [reply src/components/TodoList/TodoList.tsx:13]: agree

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**